src/mame/konami/thunderx.cpp

```diff
@@ -367,10 +367,8 @@ void thunderx_state::run_collisions( int s0, int e0, int s1, int e1, int cm, int
 {
 	uint8_t* p0;
 	uint8_t* p1;
-	int ii, jj;
 
-	p0 = &m_pmcram[16 + 5 * s0];
-	for (ii = s0; ii < e0; ii++, p0 += 5)
+	for ( p0 = &m_pmcram[s0]; p0 < &m_pmcram[e0]; p0 += 5)
```

Comment:
Please use “English” style spacing around parentheses (no space inside the parentheses) as the code was using previously.  It isn’t even consistent between the opening and closing parentheses now.

Also, you can reduce the scope of `p0` by declaring it in the `for` loop’s control expression.